bundle/language_en.properties

```diff
@@ -1737,6 +1737,8 @@ angal.userbrowser.theuseralreadyexists.fmt.msg
 angal.userbrowser.theusernamecontainsinvalidcharacters.msg                                             = The user name must contain only a-z, 0-9, dash, period and underscore characters.
 angal.userbrowser.title                                                                                = User Browser
 angal.userbrowser.user.col                                                                             = User
+angal.usergroup.permissionalreadyassigned.msg                                                          = The group permission already exists.
+angal.usergroup.permissionnotassigned.msg                                                              = Cannot delete group permission because it does not exist.
```

Comment:
I think that the key should follow exactly the translation (or viceversa)  when possible

Reply:
OK; done.